mycelo/genesis/base_config.go

```diff
@@ -147,7 +147,7 @@ func BaseConfig() *Config {
 		DowntimeSlasher: DowntimeSlasherParameters{
 			Reward:            bigIntStr("10000000000000000000"),  // 10 cGLD
 			Penalty:           bigIntStr("100000000000000000000"), // 100 cGLD
-			SlashableDowntime: 60,                                 // Should be overridden on public testnets
+			SlashableDowntime: 4,                                  // make it small so it works with small epoch sizes, e.g. 10
```

Comment:
is this desirable for every network template or only for monorepo one?

Reply:
I think so.  The default epoch size (i.e. the epoch size of the default template) is 10, so the former slashable downtime of 60 doesn't make sense with that.

The only template that has a different epoch size is the load test (1000).  If we wanted to avoid slashable downtime of 4 with epoch size 1000 it would make sense to override it on the loadtest template rather than have 60 be the default.  But the low slashable downtime makes no difference if nobody does the slashing, so I don't see it as mattering for the load test template anyway.